modules/webgl/src/adapter/resources/webgl-render-pipeline.ts

```diff
@@ -348,17 +346,12 @@ export class WEBGLRenderPipeline extends RenderPipeline {
   _areTexturesRenderable() {
     let texturesRenderable = true;
 
-    for (const [, texture] of Object.entries(this._textureUniforms)) {
-      texture.update();
-      texturesRenderable = texturesRenderable && texture.loaded;
-    }
-
     for (const [, texture] of Object.entries(this.bindings)) {
-      // texture.update();
-      // @ts-expect-error
-      if (texture.loaded !== undefined) {
-        // @ts-expect-error
-        texturesRenderable = texturesRenderable && texture.loaded;
+      if (texture instanceof WEBGLTexture) {
+        texture.update();
+        if (texture.loaded !== undefined) {
+          texturesRenderable = texturesRenderable && texture.loaded;
+        }
```

Comment:
Maybe allow texturesRenderable to be undefined or force `texture.loaded` to be a boolean, or change as suggested so we don't have to go through such hoops. 
```suggestion
      texturesRenderable = texturesRenderable && (texture.loaded || false);
```

Reply:
Done! It appears the types have become more strict since the `// @ts-expect-error` lines were originally added, and so we don't actually need to check for undefined:

```js
texturesRenderable = texturesRenderable && texture.loaded;
```

Looks OK in `webgl-texture.ts` as well.